src/azure-cli-core/azure/cli/core/auth/landing_pages/success.html

```diff
@@ -21,13 +21,5 @@
 <body>
     <h3>You have logged into Microsoft Azure!</h3>
     <p>You can close this window, or we will redirect you to the <a href="https://docs.microsoft.com/cli/azure/">Azure CLI documentation</a> in 1 minute.</p>
-    <h3>Announcements</h3>
-    <p>[Windows only] Azure CLI is collecting feedback on using the <a href="https://learn.microsoft.com/windows/uwp/security/web-account-manager">Web Account Manager</a> (WAM) broker for the login experience.</p>
-    <p>You may opt-in to use WAM by running the following commands:</p>
-    <code>
-        az config set core.enable_broker_on_windows=true<br>
-        az account clear<br>
-        az login
-    </code>
```

Comment:
These lines are introduced by https://github.com/Azure/azure-cli/pull/25416.

The `code` block in `<style>` is kept in case we need it again in the future.